packages/es-client/indexer.js

```diff
@@ -294,20 +294,6 @@ function indexReconciliationReport(esClient, payload, index = defaultIndexAlias,
   return genericRecordUpdate(esClient, payload.name, payload, index, type);
 }
```

Comment:
I'm sort of thinking we should be removing this stuff as we go. I posted a thread in Slack so curious if there's pushback here or what the reasoning would be.

Reply:
Yeah, sounds good. Thanks for flagging the test. I'll keep an eye on CI and let ya know when everything is green. The failure you caught is fixed here:

https://github.com/nasa/cumulus/pull/3760/commits/0e6c0c29624cce8380749c82cfe79036d2887cf2

I unknowingly changed the translate logic to always return a `collection` object (albeit with null properties) instead of `undefined`.